runelite-client/src/main/java/net/runelite/client/plugins/timetracking/farming/FarmingTabPanel.java

```diff
@@ -132,6 +141,9 @@ public class FarmingTabPanel extends TabContentPanel
 			}
 		}
 
+		AsyncBufferedImage compostImage = itemManager.getImage(ItemID.COMPOST);
+		compostImage.onLoaded(() ->
```

Comment:
I think I would prefer just building the compost image when the panel is updated, like weeds and the others. Also that way, you can make the image of the compost use the same compost type as is applied to the patch  (even though they items looks very similar there is a small distinction)